axelor-stock/src/main/resources/domains/PickedProduct.xml

```diff
@@ -14,9 +15,36 @@
       title="Tracking number"/>
     <many-to-one name="fromStockLocation" ref="com.axelor.apps.stock.db.StockLocation"
       title="Source location" required="true"/>
-    <decimal name="currentQty" title="Current qty" precision="20" scale="10"/>
+    <decimal name="currentQty" title="Current qty"/>
     <many-to-one name="unit" ref="com.axelor.apps.base.db.Unit" title="Unit"/>
-    <decimal name="pickedQty" title="Picked qty" required="true" precision="20" scale="10"/>
+    <decimal name="pickedQty" title="Picked qty" required="true"/>
```

Comment:
I think it was removed by mistake ? we should have this number of decimal to follow the qty scale config